protobuf-lite/src/main/java/io/grpc/protobuf/lite/ProtoLiteUtils.java

```diff
@@ -107,23 +110,38 @@ public T parse(InputStream stream) {
             }
           }
         }
+        CodedInputStream cis;
+        if (stream instanceof ReadableBuffer) {
```

Comment:
Are you guys mainly using Stubby to transfer data and information, as externally there are several technologies being used and maybe having folks focusing on using gRPC might not be a bad idea, and having them update their infrastructure.